templates/triton_services/README.ipynb

```diff
@@ -0,0 +1,334 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# Using Triton Inference Server on Anyscale Services\n",
+    "\n",
+    "**⏱️ Time to complete**: 30 minutes (1 hour on GCP)\n",
+    "\n",
+    "This guide walks you through develop and deploy Triton Server applications in Anyscale\n",
+    "though running a Stable Diffusion 1.5 service. \n",
+    "\n",
+    "In this tutorial, you will learn:\n",
+    "1. Build Docker image for Triton Server to run in Anyscale platform.\n",
+    "2. Compile model using Triton's Python backend on Anyscale Workspaces.\n",
+    "3. Run Triton Server locally on Anyscale Workspaces. \n",
+    "4. Deploy the application on Anyscale Services.\n",
+    "\n",
+    "**Note**: This guide is not meant to substitute with the official Triton documentation.\n",
+    "For more information, please refer to the\n",
+    "[Triton Inference Server documentation](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/index.html).\n",
+    "\n",
+    "## Build Docker image for Triton Server to run in Anyscale platform\n",
```

Comment:
I don't think we need to inline the Dockerfile contents here, it makes it kind of unreadable. I'd suggest:

- Add enough comments to the Dockerfile itself that it is understandable out of context
- In the body, just say "see the Dockerfile for what's required in the image"